pkg/proxy/apis/config/types.go

```diff
@@ -179,10 +179,15 @@ type KubeProxyConfiguration struct {
 	// kube-proxy is running on. If unset, the node name is assumed to be the same as
 	// the node's hostname.
 	HostnameOverride string
-	// bindAddress can be used to override kube-proxy's idea of what its node's
-	// primary IP is. Note that the name is a historical artifact, and kube-proxy does
-	// not actually bind any sockets to this IP.
-	BindAddress string
+	// nodeIPOverride can be used to override kube-proxy's idea of what its node's
+	// primary IPs are. nodeIPOverride[0] will be used for defaulting healthzBindAddress
+	// and metricsBindAddress values. If nodePortAddresses is set to "primary" node-ports
+	// will be exposed on the given ips.
+	NodeIPOverride []string
```

Comment:
I feel like we should be documenting what the node IP is for here (especially because we'll forget to change it later). We should just be documenting that this lets you override the node IP if kube-proxy is autodetecting the wrong IP(s).

Reply:
I wrote "should" when I meant "shouldn't" but you figured out what I meant anyway :slightly_smiling_face: